src/offset/local/tz_info/timezone.rs

```diff
@@ -547,48 +548,63 @@ impl fmt::Debug for TimeZoneName {
 #[derive(Debug, Copy, Clone, Eq, PartialEq)]
 pub(crate) struct LocalTimeType {
     /// Offset from UTC in seconds
-    pub(super) ut_offset: i32,
+    pub(super) ut_offset: FixedOffset,
     /// Daylight Saving Time indicator
     is_dst: bool,
     /// Time zone name
     name: Option<TimeZoneName>,
 }
 
+// Remove this hack once the MSRV >= 1.57
+macro_rules! const_unwrap {
+    ($e:expr $(,)*) => {
+        match $e {
+            ::core::option::Option::Some(x) => x,
```

Comment:
Can we do without the `::core::option::Option` verbosity? This is just an internal macro so it doesn't seem like we need it.